python-checks/src/main/resources/org/sonar/l10n/py/rules/python/S6328.json

```diff
@@ -0,0 +1,17 @@
+{
+  "title": "Replacement strings should reference existing regular expression groups",
+  "type": "BUG",
```

Comment:
Because this rule is a bug, we should also add it in the extended_ruling test . 
We can do that together!